client/src/BidderBiddingList.js

```diff
@@ -31,26 +44,51 @@ const BidderBiddingList = () => {
 		return <div>No Bidding placed yet!!</div>;
 	}
 
+	const handlePrevious = () => {
+		if (currentPage > 1) {
+			setCurrentPage(currentPage - 1);
+		}
+	};
+
+	const handleNext = () => {
+		if (currentPage < totalPages) {
+			setCurrentPage(currentPage + 1);
+		}
+	};
+
 	return (
 		<main>
 			<h1>Bidder Bidding List</h1>
 			<div className="bids-container">
-				{" "}
-				{bidderList.map((bid, index) => (
-					<div className="bid-card" key={index}>
-						<p>Status: {bid.status}</p>
+				{bidderList.map((bid) => (
+					<div className="bid-card" key={bid.bid_id}>
+						<p>Bid ID: {bid.bid_id}</p>
+						<p>Tender ID: {bid.tender_id}</p>
+						<p>Tender Title: {bid.title}</p>
 						<p>
-							submitted on: {new Date(bid.submission_date).toLocaleDateString()}
+							Tender Closing Date:{" "}
```

Comment:
```suggestion
							Tender Closing Date:
```

Remove all uses of `{" "}`, it's unnecessary